### PR TITLE
Behaviors: Move behaviors code to experimental folder.

### PR DESCRIPTION
**Pull request by cbravobernal**
## What?

Behaviors are not getting to 6.3. Due to that reason, the code should still be in experimental.

### CODE REVIEW

File: lib/load.php [hunk 1]
```diff
@@ -97,6 +96,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 
 // Experimental features.
 remove_action( 'plugins_loaded', '_wp_theme_json_webfonts_handler' ); // Turns off WP 6.0's stopgap handler for Webfonts API.
+require __DIR__ . '/experimental/behaviors.php';
```
Comment (artemiomorales) [inline lib/load.php:99]: Should the comment `Turns off WP 6.0's stopgap handler for Webfonts API` be updated or removed?

**Review by artemiomorales — COMMENTED** [review]
I don't know enough about how the `experimental/` versus `compat/` folders work and their implications to offer an approving review, though I can say that I've tested the lightbox functionality and all seems to working as expected 👍

**Comment by ramonjd** [comment]
This makes sense to me. 

PHP changes in `/compat/x-y` folders correspond to features that have been synched to the `x.y` releases of WordPress.

It means, for example, that folks who are still running, say, WordPress 6.2, can still use 6.3-compatible features if they have the Gutenberg plugin installed. It also acts as a guide for us so that we have a good idea about which features have made it to Core and which haven't 😄 

More info over at the README: 

https://github.com/WordPress/gutenberg/blob/32d394c3518fccdc8ce57d8d3ed8f0ff8cf33ff3/lib/README.md#L1-L1

So if the changes in this PR are not being ported to WordPress 6.3 then I think `/experimental` is a valid place for them to live until they're stable/earmarked for a WordPress release.

Thanks!

**Review by SantosGuillamot — APPROVED** [review]
Looks good to me 🙂

**Pull request merged**